sysboundary/ionosphere.cpp

```diff
@@ -1313,6 +1314,9 @@ namespace SBC {
 
                  // Local cell
                  std::array<int,3> lfsc = technicalGrid.globalToLocal(fsc[0],fsc[1],fsc[2]);
+                 if(lfsc[0] == -1 && lfsc[1] == -1 && lfsc[2] == -1) {
+                    continue;
+                 }
```

Comment:
I would actually suggest that these should be || instead of &&.

Reply:
Actually, I'll just fix that myself after merging :)

Reply:
Yeah but fsgrid returns the triplet in any case, not individually. Hence this choice. Checking the first would be enough, as you did in other spots.